black-mulberry-core/src/main/java/com/black/mulberry/core/util/UserServiceUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.black.mulberry.core.util;
+
+import com.black.mulberry.core.entity.User;
+import com.black.mulberry.core.exception.UserNotFoundException;
+import com.black.mulberry.core.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class UserServiceUtil {
+    private final UserRepository userRepository;
```

Comment:
why its util? in general ist ok, buts it can be better - lets rename UserSServiceUtil to UserServiceSupport in package ......service.support.UserServiceSupport.java and we can change @Component anotation to @Service.